R/read-data.R

```diff
@@ -22,6 +22,8 @@ read_population <- function(file) {
   val <- read_csv_partial_cols(file, col_types = col_spec)
   readr::stop_for_problems(val)
 
+  val <- drop_na_rows(val)
```

Comment:
Just looking at this there is quite a lot of code duplication. I'd be tempted to move 23-29, 61-67, 94-100 and 149-155 into `read_csv_partial_cols`

Reply:
I was also tempted to do this when implementing. On the other hand, if felt like `read_csv_partial_col()` does 'a thing', and wasn't sure about extending it.  But if you and I had the same instinct...should do it...